From 7978bed4d52e44feae114ba56e9b5035b7dd2c1c Mon Sep 17 00:00:00 2001
From: Rich Megginson <rmeggins@redhat.com>
Date: Wed, 17 Apr 2024 10:14:21 -0600
Subject: [PATCH 105/115] chore: change no_log false to true; fix comment

Forgot to change a `no_log: false` back to `no_log: true` when debugging.
Fix an error in a comment

Signed-off-by: Rich Megginson <rmeggins@redhat.com>
(cherry picked from commit b37ee8fc7e12317660cca765760c32bd4ba91035)
---
 tasks/handle_secret.yml | 2 +-
 vars/main.yml           | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tasks/handle_secret.yml b/tasks/handle_secret.yml
index b3677ef..02bc15b 100644
--- a/tasks/handle_secret.yml
+++ b/tasks/handle_secret.yml
@@ -39,7 +39,7 @@
   become: "{{ __podman_rootless | ternary(true, omit) }}"
   become_user: "{{ __podman_rootless | ternary(__podman_user, omit) }}"
   when: not __podman_rootless or __podman_xdg_stat.stat.exists
-  no_log: false
+  no_log: true
   vars:
     __params: |
       {% set rc = {} %}
diff --git a/vars/main.yml b/vars/main.yml
index 47293c5..38402ff 100644
--- a/vars/main.yml
+++ b/vars/main.yml
@@ -74,5 +74,5 @@ __podman_user_kube_path: "/.config/containers/ansible-kubernetes.d"
 # location for system quadlet files
 __podman_system_quadlet_path: "/etc/containers/systemd"
 
-# location for user kubernetes yaml files
+# location for user quadlet files
 __podman_user_quadlet_path: "/.config/containers/systemd"
-- 
2.46.0

